docs/_includes/listbox-button.html

```diff
@@ -190,7 +196,10 @@ <h3 id="listbox-button-selected">Selected Listbox Button</h3>
             <span class="listbox-button" data-makeup-auto-select="false">
                 <button class="btn btn--form" style="min-width: 150px" aria-expanded="false" aria-haspopup="listbox" data-listbox-button-prefix="Color: ">
                     <span class="btn__cell">
-                        <span class="btn__text">Color: Blue</span>
+                        <span class="btn__contents">
```

Comment:
Is there a reason why we have the label and text nested under contents? Does it not look correct? Is there a way we can achieve it with 1 layer?

Just wondering because if we don't need an extra layer, then let's not add it. IMO its cleaner to have only 1 layer, especially if it's not needed. .

Reply:
I was debating this very thing. I ended up adding the layer as a logical grouping of the related (key:value) elements. It's not necessary. I guess I'll flatten it, but It will need some flex tweaking too.

Reply:
So an easy way to get the flex to work is add these styles to the `btn__text` (again this is probbly only for listbox and if a `btn__label` preceeds)
```
text-align: left;
flex: 1;
margin-left: 3px; // I picked a random value, but if you don't have margin it seems a bit close to the label
```

Yeah I think it's much cleaner having a flattened structure.

Reply:
Too late; I already changed it. Yea, wasn't too bad.